code/PostProcessing/TriangulateProcess.cpp

```diff
@@ -468,6 +468,20 @@ bool TriangulateProcess::TriangulateMesh( aiMesh* pMesh)
                         continue;
                     }
 
+                    // Skip when three point is in a line
+                    aiVector2D left = *pnt0 - *pnt1;
+                    aiVector2D right = *pnt2 - *pnt1;
+
+                    left.Normalize();
+                    right.Normalize();
+                    auto mul = left * right;
+
+                    // if the angle is 0 or 180
+                    if (std::abs(mul - 1.f) < ai_epsilon || std::abs(mul + 1.f) < ai_epsilon) {
+                        // skip this ear
```

Comment:
This should log a warning so user knows the model is potentially broken.

Reply:
done